pocketimdb/movies/paginations.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.pagination import PageNumberPagination
+from rest_framework.response import Response
+
+DEFAULT_PAGE = 1
+
+class CustomPageNumberPagination(PageNumberPagination):
+    page = DEFAULT_PAGE
+    page_size = 10
+
+    def get_paginated_response(self, data):
+        return Response({
+            'total': self.page.paginator.count,
+            'page': int(self.request.GET.get('page', DEFAULT_PAGE)),
+            'page_size': self.page_size,
+            'results': data
+        })
```

Comment:
ovdje mozemo dodati 
```
'total_pages', self.page.paginator.num_pages
```

i onda na FE ne moras vrsitit ovu kalkulaciju i odmah ces znati koliko ima stranica

Reply:
Ovo je bas korisno, dodacu sad to, nisam znao da postoji taj property u paginator-u.